Emby.Server.Implementations/Networking/NetworkManager.cs

```diff
@@ -245,23 +240,57 @@ public bool IsAddressInSubnets(string addressString, string[] subnets)
             return IsAddressInSubnets(IPAddress.Parse(addressString), addressString, subnets);
         }
 
+        // returns true if address is in the LAN list in the config file
```

Comment:
This should be moved to `INetworkManager` and properly formatted as documentation. See [https://docs.microsoft.com/en-us/dotnet/csharp/codedoc](url) if you don't know how.